sqlglot/tokens.py

```diff
@@ -111,6 +111,7 @@ class TokenType(AutoName):
 
     # keywords
     ALIAS = auto()
+    ALTER_TABLE = auto()
```

Comment:
i thought you were going to delete this?

Reply:
Done